Userland/Libraries/LibJS/Bytecode/Op.h

```diff
@@ -541,6 +541,29 @@ class GetById final : public Instruction {
     IdentifierTableIndex m_property;
 };
 
+class GetByIdWithThis final : public Instruction {
```

Comment:
Before breaking the optimized mode again, make sure to add this to the `EliminateLoads` pass under `GetById`
same blow